packages/near-fast-auth-signer/src/utils/multi-chain/chains/types.ts

```diff
@@ -0,0 +1,68 @@
+import { KeyPair } from '@near-js/crypto';
+
+export type ChainSignatureContracts = 'multichain-testnet-2.testnet'
+
+export type UTXO = {
+  txid: string;
+  vout: number;
+  value: number
+}
+
+type BaseTransaction = {
+  to: string;
+  value: string;
+  derivedPath: string
+};
+
+export type EVMTransaction = BaseTransaction & ({
+  gasLimit: bigint;
+  maxFeePerGas: bigint;
+  maxPriorityFeePerGas: bigint;
+} | {
+  gasLimit?: never;
+  maxFeePerGas?: never;
+  maxPriorityFeePerGas?: never;
+});
+
+type BtcInputsAndOutputs = {
+  inputs: UTXO[];
+  outputs: { address: string, value: number }[];
+};
+
+export type BTCTransaction = BaseTransaction & (BtcInputsAndOutputs | {
+  inputs?: never;
+  outputs?: never;
+});
+
+type ChainProviders = {
```

Comment:
We’ll add this in next iteration because currently we still don’t support not having a passkey FAK in the browser and trying to use sign or sign multi chain in the signer app